sdk/include/reactos/x86x64/Cpuid.h

```diff
@@ -0,0 +1,205 @@
+
```

Comment:
I would be interesting to have a license header in that file (and in the other Msr.h file), as it is done in those other Amd and Intel files.